Use busybox wget

brknkfr opened the pull request:
- No need for real wget anymore, newer busybox wget works just fine with https urls
- Add needed libraries for resolving

Commit:
Use busybox wget

- No need for real wget anymore, newer busybox wget works just fine with https urls
- Add needed libraries for resolving

malvinas2 commented:
I didn't understand at first why one dependency (wget) should be replaced by another (busybox).
But if I understand it correctly, busybox is also used for initram anyway?

brknkfr commented:
Yes, at least on debian, initramfs-tools-core installs busybox anyway ...

Commit:
Add new dependencies
- Require at least busybox 1.3 which supports https in wget
- Require busybox, because busybox-static can cause troubles (https://sourceware.org/bugzilla/show_bug.cgi?id=17250)
